src/Perspex.Controls/Shapes/Shape.cs

```diff
@@ -90,13 +90,20 @@ public double StrokeThickness
             set { SetValue(StrokeThicknessProperty, value); }
         }
 
+        public PenLineCap DashCap { get; set; } = PenLineCap.Flat;
```

Comment:
These properties in WPF are name `StrokeDashCap`, `StrokeStartLineCap` etc - we should probably use the same naming convention here.